sample.env

```diff
@@ -7,7 +7,7 @@ MNEMONIC="deputy annual debris outer baby prefer mammal scene insect obtain parr
 DEPLOYER=0xff22437ccbedfffafa93a9f1da2e8c19c1711052799acf3b58ae5bebb5c6bd7b
 
 MAINNET_DEPLOYMENT_SALT=0
-TESTNET_DEPLOYMENT_SALT=2
+TESTNET_DEPLOYMENT_SALT=3
```

Comment:
Do we have to update the sample salt? Was this on accident and you meant to update your own .env file?